data/data/digital-objects-per-organ.vl.json

```diff
@@ -15,16 +18,33 @@
       "labelColor": "#4B4B5E"
     },
     "legend": {
-      "labelFont": "Metropolis",
       "labelFontSize": 14,
-      "labelFontWeight": 500,
-      "labelColor": "#201E3D"
+      "titleFontSize": 14,
+      "titleLineHeight": 21,
+      "titleColor": "#201E3D",
+      "labelColor": "#4B4B5E",
+      "titleFontWeight": 500,
+      "labelFontWeight": 500
     }
   },
   "mark": { "type": "point", "tooltip": true, "filled": true },
   "encoding": {
-    "x": { "field": "organ", "type": "nominal", "title": "Organ" },
-    "y": { "field": "type", "type": "nominal", "title": "Digital Object (DO) Type" },
+    "x": {
+      "field": "organ",
+      "type": "nominal",
+      "title": "Organ",
+      "axis": {
+        "labelAngle": -45
```

Comment:
Where there any problems with the labels not showing/getting out of bounds?

Reply:
When I checked on the online editor, the labels on x axis were very close to each other, but did not go out of bounds. need to check how it looks in 3/1 aspect ratio